README.md

```diff
@@ -181,6 +181,90 @@ Tested on Ubuntu 16.04/18.04/20.04, macOS 10.15, Windows 10, and Windows Subsyst
 
 ### Minimal Examples
 
+<details>
+<summary>CustomRunner – PyTorch for-loop decomposition</summary>
+<p>
+
+```python
+import os
+from torch import nn, optim
+from torch.nn import functional as F
+from torch.utils.data import DataLoader
+from catalyst import dl, metrics
+from catalyst.data.transforms import ToTensor
+from catalyst.contrib.datasets import MNIST
+
+model = nn.Sequential(nn.Flatten(), nn.Linear(28 * 28, 10))
+optimizer = optim.Adam(model.parameters(), lr=0.02)
+
+loaders = {
+    "train": DataLoader(
+        MNIST(os.getcwd(), train=True, download=True, transform=ToTensor()), batch_size=32
+    ),
+    "valid": DataLoader(
+        MNIST(os.getcwd(), train=False, download=True, transform=ToTensor()), batch_size=32
+    ),
+}
+
+class CustomRunner(dl.Runner):
+    def predict_batch(self, batch):
+        # model inference step
+        return self.model(batch[0].to(self.device))
+
+    def on_loader_start(self, runner):
+        super().on_loader_start(runner)
+        self.meters = {
+            key: metrics.AdditiveValueMetric(compute_on_call=False)
+            for key in ["loss", "accuracy01", "accuracy03"]
+        }
```

Comment:
What about removing `on_loader_start`/`on_loader_end` methods? I think it can make the example a little bit simpler?